### PR TITLE
Continuous Integration (CI) (ver.2)

### PR DESCRIPTION
**Pull request by XuMingSheng**
- rspec and cucumber
- rubocop checks
  - The configuration `.rubocop.yml` is still a draft and messy for now. 
  - Skpping test files for now
  - Not adding auto-commit because that may cause conflicts
 - Revise some contents in docker-compose files.

### CODE REVIEW

File: .github/workflows/rubocop-checks.yml
Comment (neilgautam24-tamu) [inline .github/workflows/rubocop-checks.yml:1]: Remove commented Code

Comment (neilgautam24-tamu) [inline .github/workflows/rubocop-checks.yml:1]: Remove Commented Code in this file

**Review by neilgautam24-tamu — CHANGES_REQUESTED** [review]

File: .github/workflows/rubocop-checks.yml
Comment (XuMingSheng) [reply .github/workflows/rubocop-checks.yml:1]: removed

Comment (XuMingSheng) [reply .github/workflows/rubocop-checks.yml:1]: removed

**Review by neilgautam24-tamu — APPROVED** [review]

**Pull request merged**